modules/polls/components/PollVoteHistoryItem.tsx

```diff
@@ -0,0 +1,86 @@
+/** @jsx jsx */
+import VoteBreakdown from './VoteBreakdown';
+import Tooltip from 'components/Tooltip';
+import { getNetwork } from 'lib/maker';
+import moment from 'moment';
+import Link from 'next/link';
+import { Box, Text, jsx, Link as ThemeUILink } from 'theme-ui';
+import { PollVoteHistory } from '../types/pollVoteHistory';
+import { PollVoteResultsCompact } from './PollVoteResultsCompact';
+import { parseRawPollTally } from '../helpers/parseRawTally';
+
+export function PollVoteHistoryItem({ vote }: { vote: PollVoteHistory }): React.ReactElement {
```

Comment:
It looks like we're missing the Discussion link here from the design.

Reply:
I know, is not finished. Will be added in new PRs

Reply:
yes, will be added